meteor/sofie-server/business-logic/services/rundown-event-service.ts

```diff
@@ -0,0 +1,24 @@
+import { RundownEventListener } from './interfaces/rundown-event-listener'
+import { RundownEvent } from '../../model/interfaces/rundown-event'
+import { RundownEventEmitter } from './interfaces/rundown-event-emitter'
+
+export class RundownEventService implements RundownEventEmitter, RundownEventListener {
+	private static instance: RundownEventService
+
+	public static getInstance(): RundownEventService {
+		if (!this.instance) {
+			this.instance = new RundownEventService()
+		}
+		return this.instance
+	}
```

Comment:
This can be updated to the following:
`public static instance: RundownEventService => new RundownEventService()`

Typescript/Node is clever enough to only create one instance.
This was something @KvelaGorrrrnio originally suggested to me, and i was sceptical about, so i tested it.
I have also been using the same structure as this for creating Singletons, but thanks to Anders, have learned that the suggested change works in Typescript and C#.

Anders, do fell free to correct me, if i made some error in my explanation.